pkg/events.go

```diff
@@ -210,6 +213,84 @@ func (octopus *EventOctopus) Unsubscribe(service, subject string) error {
 	return nil
 }
 
+type dbDiagnosticResult struct {
+	connectionString string
```

Comment:
I also mentioned this in the other PR, should we log the connection string risking that we log database credentials (specified inside the connstring?)

Reply:
removed